lib/commands/context.js

```diff
@@ -192,6 +194,12 @@ async function setupExistingChromedriver (chromedriver) {
 }
 
 async function setupNewChromedriver (opts, curDeviceId, adbPort) {
+  // if a port wasn't given, pick a random available one
+  if (opts.chromeDriverPort === null || opts.chromeDriverPort === undefined) {
```

Comment:
I used `!opts.chromeDriverPort`. Tested in node and it works for both undefined and null fields in an object.